com.github.vladimiry.ElectronMail.metainfo.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
```suggestion
<component type="desktop-application">
```

Also this file could be upstreamed if possible, it's not specific to Flathub.

Reply:
I just removed this file since it's not required for the app to be submitted (not specific to Flathub).

Reply:
The file is a requirement, just not having it here on Flathub is not.

Reply:
Can you point me to the guide where it's described? Asking since I see for example this app https://github.com/flathub/com.github.unrud.VideoDownloader doesn't include the metadata files.

Reply:
Metadata file removing got reverted in 998b9c341d039bab35ecc7a0eb95230f29911606. I'd prefer the file to be hosted here on flathub repo.